lib/Ohmbrewer_Thermostat.cpp

```diff
@@ -323,7 +331,7 @@ int Ohmbrewer::Thermostat::doDisplay(Ohmbrewer::Screen *screen) {
     // Print the section title
     screen->print("== Thermostat #");
     screen->print(getID());
-    screen->println(" ==");
+    screen->print("  ==");
```

Comment:
does this work? I had to switch this to println becasue a print margin didnt automatically make a new line.